pkgs/by-name/ho/hof/package.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
```

```suggestion
, fetchFromGitHub
, installShellFiles
```
needed for shell completions